lib/index.js

```diff
@@ -187,17 +187,17 @@
     render() {
       var self;
       self = this;
-      console.log("Generating Template Files.....");
+      if( !self.options.verbose ) console.log("Generating Template Files.....");
```

Comment:
Shouldn't this (and the ones below) be `if(self.options.verbose) console.log("Generating Template Files.....");` instead?